### PR TITLE
Deep linking E2E spec

### PR DESCRIPTION
**Pull request by cortisiko**
**Description**

Adding deep linking coverage to our E2E test suite. 

This is 🚫 🚫 🚫 by https://github.com/MetaMask/metamask-mobile/pull/3650
**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

File: app/components/Views/ImportFromSeed/index.js [hunk 1]
```diff
@@ -408,6 +408,7 @@ class ImportFromSeed extends PureComponent {
 					style={styles.biometrySwitch}
 					trackColor={Device.isIos() ? { true: colors.green300, false: colors.grey300 } : null}
 					ios_backgroundColor={colors.grey300}
+					testID={'remember-me-toggle'}
```
Comment (gantunesr) [inline app/components/Views/ImportFromSeed/index.js:411]: Do we have a standard for the `testID` names?

File: e2e/pages/Onboarding/ImportWalletView.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ const CREATE_PASSWORD_INPUT_BOX_ID = 'input-password-field';
 const CONFIRM_PASSWORD_INPUT_BOX_ID = 'input-password-field-confirm';
 const IOS_I_UNDERSTAND_BUTTON_ID = 'password-understand-box';
 const ANDROID_I_UNDERSTAND_BUTTON_ID = 'i-understand-text';
+const REMEBER_ME_ID = 'remember-me-toggle';
```
Comment (gantunesr) [inline e2e/pages/Onboarding/ImportWalletView.js:10]: typo
```suggestion
const REMEMBER_ME_ID = 'remember-me-toggle';
```

Comment (cortisiko) [reply e2e/pages/Onboarding/ImportWalletView.js]: hawk eye. Fixed!

File: e2e/pages/Onboarding/ImportWalletView.js [hunk 3]
```diff
@@ -53,6 +54,10 @@ export default class ImportWalletView {
 		await TestHelpers.clearField(SECRET_RECOVERY_PHRASE_INPUT_BOX_ID);
 	}
 
+	static async toggleRememberMe() {
+		await TestHelpers.tap(REMEBER_ME_ID);
```
[the remark this answers is not in the document]
Comment (cortisiko) [reply e2e/pages/Onboarding/ImportWalletView.js]: hawk eye. Fixed!

File: e2e/specs/deeplinks.spec.js [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+const BINANCE_RPC_URL = 'https://bsc-dataseed1.binance.org';
+const POLYGON_RPC_URL = 'https://polygon-rpc.com/';
+
+const binanceDeepLink = 'https://metamask.app.link/send/0xB8B4EE5B1b693971eB60bDa15211570df2dB228A@56?value=1e14';
+
+const polygonDeepLink =
+	'https://metamask.app.link/send/0x0000000000000000000000000000000000001010@137/transfer?address=0xC5b2b5ae370876c0122910F92a13bef85A133E56&uint256=3e18';
+
+const ethereumDeepLink = 'https://metamask.app.link/send/0x1FDb169Ef12954F20A15852980e1F0C122BfC1D6@1?value=1e13';
+const rinkebyDeepLink = 'https://metamask.app.link/send/0x1FDb169Ef12954F20A15852980e1F0C122BfC1D6@4?value=1e13';
```
Comment (gantunesr) [inline e2e/specs/deeplinks.spec.js:30]: Is there a possibility of these links being unavailable and causing the e2e to fail?

File: e2e/specs/deeplinks.spec.js [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+			await TestHelpers.checkIfNotVisible('onboarding-wizard-step1-view');
+		} catch (e) {
+			//
+		}
+	});
+
+	// it('should attempt to deep link to the send flow with a custom network not added to wallet', async () => {
+	// 	await TestHelpers.openDeepLink(binanceDeepLink);
+	// 	await TestHelpers.delay(3000);
+	// });
```
Comment (gantunesr) [inline e2e/specs/deeplinks.spec.js:79]: Is there a reason to keep this?

File: e2e/specs/deeplinks.spec.js (hunk 4 above)
Comment (cortisiko) [reply e2e/specs/deeplinks.spec.js:30]: Yeah this PR depends on #3650. As of right now some of the tests are going to fail.

File: e2e/specs/deeplinks.spec.js (hunk 5 above)
Comment (cortisiko) [reply e2e/specs/deeplinks.spec.js:79]: Yeah once we get the 👍 on #3650 I am going to uncomment/remove based on feedback

**Review by gantunesr — APPROVED** [review]
LGTM

**Pull request merged**